README.md

```diff
@@ -519,6 +519,11 @@ The `pointcloud_postgis` extension adds functions that allow you to use PostgreS
 > 
 >     POINT Z (-127 45 124)
 
+**PC_Envelope(pcpatch)** returns **geometry**<br/>
```

Comment:
Yes. We should explicitly say that the return value is a PostGIS 2D Polygon representing a box that bounds the patch along the X and Y axis. Feel free to change the wording though :)